packages/browser-integration-tests/src/segment-retries.test.ts

```diff
@@ -0,0 +1,64 @@
+import { test, expect } from '@playwright/test'
+import { SettingsBuilder } from './fixtures/settings'
+import { standaloneMock } from './helpers/standalone-mock'
+
+test.describe('Standalone tests', () => {
+  test.beforeEach(standaloneMock)
+
+  test.describe('Segment.io Retries', () => {
+    test.beforeEach(async ({ context }) => {
+      await context.route(
+        'https://cdn.segment.com/v1/projects/*/settings',
+        (route, request) => {
+          if (request.method().toLowerCase() !== 'get') {
+            return route.continue()
+          }
+
+          return route.fulfill({
+            status: 200,
+            headers: {
+              'Content-Type': 'application/json',
+            },
+            body: JSON.stringify(new SettingsBuilder().build()),
+          })
+        }
+      )
+    })
+
+    test.skip('supports retries on page navigation', async ({ page }) => {
```

Comment:
Did you mean to leave this skip in?

Reply:
Yes - I wrote this test for #714 and verified locally that this test passes once #714 is merged. It's functionality that should work today but doesn't. Basically once this gets merged in I'll rebase #714 and remove the `skip` here. I left it in this PR just so that if anything were to happen the test case wouldn't be lost to time :grin: